recipes/libnghttp3/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
```

Comment:
Please, remove the entire test_v1_package folder, it's no longer needed: https://github.com/conan-io/conan-center-index/discussions/17406

Reply:
Thanks for the review! I used libnghttp2 as a template for this recipe since both projects have the same authors and fall under the same administrative umbrella. Many of the issues you have pointed out appear in that recipe as well.

I'll post and update with these changes shortly!